package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "description": "AsyncAPI website",
   "scripts": {
-    "dev": "node scripts/build-post-list.js && node scripts/build-rss.js && node scripts/build-roadmap.js && next -p 8080",
-    "build": "node scripts/build-post-list.js && node scripts/build-rss.js && node scripts/build-roadmap.js && next build",
+    "dev": "node scripts/build-doc-list.js --output=posts.json && node scripts/build-rss.js && node scripts/build-roadmap.js && next -p 8080",
+    "build": "node scripts/build-doc-list.js --output=posts.json && node scripts/build-rss.js && node scripts/build-roadmap.js && next build",
```

Comment:
Can we start splitting this into different npm scripts? I'd create `build:posts`, `build:rss`, and `build:roadmap`.

Reply:
Yeah sounds like a plan